recipes/gtabview/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "gtabview" %}
+{% set version = "0.8" %}
+{% set hash_type = "sha256" %}
+{% set hash = "959160e6ee65110fb43af8fdc7e83cbb7469b91f90ab25b63c611241c7bf5fb1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  entry_points:
+    - gtabview = bin.cli.main:main
```

Comment:
I would rather have the cli script be inside the actual package and explicitly reference it instead of moving stuff around, so I am waiting on the package maintainers to provide some input